index.bs

```diff
@@ -392,7 +394,12 @@ Issue: Links for image features are broken, need to actually define that and lin
 
 Security and Privacy considerations {#privacy}
 ===========
-See [[!RFC8942]].
+
+This specification exposes information regarding the user's preferences and agent, which can be used as an active fingerprinting vector.
+[=User agents=] implementing this specification need to be aware of that, and take that into consideration when deciding whether to implement specific hints,
+lie in the value returned for a given hint, or ommit the hint entirely.
+
+For example, [=user agents=] may decide to omit or lie about the user's preferences when the user is in a private browsing mode.
```

Comment:
Not sure that this specific example makes sense, as it can help detect private browsing mode, which is also not great.